catalog/ui/search-ui/standard/src/main/webapp/js/controllers/cesium.controller.js

```diff
@@ -59,10 +62,25 @@ define(['application',
                 this.listenTo(wreqr.vent, 'metacard:overlay', this.overlayImage);
                 this.listenTo(wreqr.vent, 'metacard:overlay:remove', this.removeOverlay);
 
+                this.clustering = false;
+
+                $(".cesium-viewer-toolbar").append("<button class='cesium-button cesium-toolbar-button cesium-home-button'><span class='cluster-results'></span></button>");
```

Comment:
If Cesium is not reloaded but this controller is, you could get multiple buttons added since it will keep appending.  

In general we try to avoid global use of jQuery.  Ideally the view that contains Cesium would manage this and use `this.$('.cesium-viewer-toolbar')` instead.